dev-env/README.md

```diff
@@ -106,14 +106,16 @@ To **test** your plugin (a.k.a. `PluginDefinition` yaml file) with a backend (a.
 - Deploy your `PluginDefinition` to the `dev-env` with the reference to your helm chart in `Spec.helmChart`
 - Onboard a cluster you have access to (e.g. local [minikube](https://minikube.sigs.k8s.io/docs/start/), [KIND](https://kind.sigs.k8s.io/docs/user/quick-start/) or other)
```

Comment:
And maybe we should also refer to https://github.com/cloudoperators/greenhouse/pull/261/files ?

Reply:
It's linked the other way around: https://github.com/cloudoperators/greenhouse/pull/261/files#diff-512b41f63c2f59d7ec040981591106d11dd81a8f5f3ab9b8f868b9bf06bfe0a5R33

Reply:
I saw :)
But it might be valid to have a hop off point from the dev-env Readme, too?

Reply:
Maybe also removing the whole cluster onboarding section from here and just reference the user guide. No need for duplication.

Reply:
The Guide is intended as a general starting point that points to the relevant docs for the local environment. 
Maybe it would make sense to move the dev-env readme into an own subfolder within the Website that covers gives an intro into UI development, Controller Development against the dev-env, as well as the guide that covers deploying a plugindefinition.
But that is out of scope for this PR.